x-pack/plugins/ml/server/models/file_data_visualizer/file_data_visualizer.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import Boom from 'boom';
+import fs from 'fs';
+import os from 'os';
+const util = require('util');
+// const readFile = util.promisify(fs.readFile);
```

Comment:
Is this commented out line still needed?

Reply:
i'm leaving this commented code in as this is a work in progress and will eventually be used.